docs/parameters/ms.session.key.field.md

```diff
@@ -25,14 +25,15 @@ a session in backend by a status field, a session cursor, or through
 `ms.session.key.field` specifies the key field in response in order to retrieve the 
 status for session control and the condition for termination.
 
-This property takes the form a JsonObject with a **name**, a **condition**, 
-and a **failCondition**.
+This property takes the form a JsonObject with a **name**, a **condition**, a **failCondition**,
+and a **initValue**.
 
 it takes the form a Json object with a "name", "condition", and "failCondition".
 - **name** specifies the field in response that gives session info, it is required
 - **condition** specifies when the session should stop.
 - **failCondition** specifies when the session should fail.
-- "condition" and "failCondition" are optional
+- **initValue** specifies the initial value of the session key.
```

Comment:
Can you provide some more details on the format of `initValue`? Should we expect a string primitive or json, etc.?

Reply:
Sure thing, updated